django/db/backends/base/features.py

```diff
@@ -339,6 +339,15 @@ class BaseDatabaseFeatures:
     # Does the backend support column comments in ADD COLUMN statements?
     supports_comments_inline = False
 
+    # Does the backend support generated columns?
+    supports_generated_columns = False
```

Comment:
We might be able to remove this and only have `supports_stored_generated_columns` and `supports_virtual_generated_columns`. I'm not certain that would be better though.